bert_e/git_host/github/__init__.py

```diff
@@ -501,6 +507,58 @@ def __str__(self) -> str:
         return self.state
 
 
+class AggregatedCheckSuites(base.AbstractGitHostObject,
+                            base.AbstractBuildStatus):
+    """
+    The Endpoint to have access infos about github actions CI
+    """
+    GET_URL = '/repos/{owner}/{repo}/commits/{ref}/check-suites'
+    SCHEMA = schema.AggregateCheckSuites
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._check_suites = [elem for elem in self.data['check_suites']]
+
+    @property
+    def state(self):
+        queued = any(
+            elem['status'] == 'queued' for elem in self._check_suites
+        )
+        all_complete = all(
+            elem['status'] == 'completed' for elem in self._check_suites
+        )
+        all_success = all(
+            elem['conclusion'] == 'success' for elem in self._check_suites
+        )
+        if self._check_suites.__len__() > 0 and queued:
+            return 'NOTSTARTED'
+        elif self._check_suites.__len__() > 0 and not all_complete:
+            return 'INPROGRESS'
+        elif self._check_suites.__len__() > 0 and all_complete and all_success:
+            return 'SUCCESSFUL'
+        else:
+            return 'FAILED'
+
+    @property
+    def url(self):
+        return ''
```

Comment:
TODO: here we need to return a url that can be used in the queue status page.

Is there a url we can build on a sha1 or branch name that can display in github actions all the workflow that ran for a single branch?

Reply:
Noted as you found out that if you can return the following url here https://github.com/scality/bert-e/runs/2529098077 it would be perfect.